.github/workflows/test.yml

```diff
@@ -32,4 +32,4 @@ jobs:
           python -m pip install ".[test]"
       - name: Run tests
         run: |
-          python -m pytest
+          python -m pytest tests
```

Comment:
If you don't want to change the way pytest is called you can add the following to pyproject.toml in the tool.pytest.ini.options section:

`addopts = "--ignore=third_party"
`

Reply:
You are right, this should probably go into `pyproject.toml` as it's a good idea locally too. 
I'll specify `testpaths` there